packages/code-du-travail-frontend/src/annuaire/adresse.service.js

```diff
@@ -0,0 +1,55 @@
+import getConfig from "next/config";
+import memoizee from "memoizee";
+
+const {
+  publicRuntimeConfig: { API_URL }
+} = getConfig();
+
+function search(query) {
+  const url = `https://api-adresse.data.gouv.fr/search/?q=${encodeURIComponent(
+    query
+  )}&type=housenumber&limit=5`;
+  return fetch(url).then(response => {
```

Comment:
Me might want a wrapper for the redondant pattern

```js
return fetch(url).then(response => {
    if (!response.ok) {
      throw new FetchError(...)
    }

    return response.json()
}
```

[Can be done in another PR]